.github/workflows/maven.yml

```diff
@@ -85,6 +86,7 @@ jobs:
     needs: package-and-upload
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
TODO: consider removing the `fail-fast: false` rule for each of these.
On the one hand it makes failing PRs fail quicker.
On the other hand it means going through several (long) iterations before we can tell what all is broken.

Reply:
Leaving this as is for now.